### DISCUSSION TITLE
Bug in Map (Wiesbaden/Germany)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#16980] [Maps Related]
There is a bug in map at position 50.04238° N, 8.21260° E
The Highway has a "gap" here. I checked the data at OSM: there it is correct. I downloaded newest live-Update some minutes ago... Same gap... 
![Screenshot_20230415-192240](https://user-images.githubusercontent.com/62500258/232242714-337d8621-646f-4283-9304-17220bfd542d.png)

**Comment by ThomasMeister123** [comment]
Still not fixed in todays Update ;(
![Screenshot_20230416-143739](https://user-images.githubusercontent.com/62500258/232311878-f00add93-be5c-4a44-9231-4e0982697c69.png)

**Comment by scaidermern** [comment]
Could be the same issue as #16939.

For Bugs it is better to open a ticket (or comment on an existing one) instead of creating a discussion.

**Reply by ThomasMeister123** [reply]
Disappointing that the problem is still not solved. 😟
